docs/orchestration/concepts/artifacts.md

```diff
@@ -0,0 +1,88 @@
+# Artifacts <Badge text="Beta"/>
+
+The [Artifacts API](/api/latest/backend/artifacts.html) enables you to publish data from task runs that is rendered natively in the Prefect UI, both Prefect Cloud and Prefect Server. 
+
+Using the Artifacts API, you can easily publish information directly to the Prefect UI. These published artifacts are linked to specific task runs and flow runs, and artifacts can render more sophisticated information than you'd typically log, such as reports, tables, charts, images, and links to external data.
+
+Currently, the Artifacts API enables you to render the following artifact types:
+
+- Links
+- Markdown
+
+Link artifacts render a clickable hyperlink as an artifact.
+
+Markdown artifacts render strings that can include [Github-flavored Markdown](https://github.github.com/gfm/) markup for formatting. 
+
+::: tip Artifacts render individually
+Note that each artifact you create in a task renders as an individual artifact in the Prefect UI. In other words, each call to `create_link()` or `create_markdown()` creates its own, separate artifact.
+
+Within a task, you may use these commands as many times as necessary, but they do not operate in the same manner as a `print()` command where you might string together multiple calls to add additional items to a report. 
+
+As a best practice, such as when using `create_markdown()` to create artifacts like reports or summaries, compile your message string separately, then pass to `create_markdown()` to create the full artifact.
+:::
+
+For more background on the design goals for the Artifacts API, see the [Introducing: The Artifacts API](https://medium.com/the-prefect-blog/introducing-the-artifacts-api-b9e5972db043) blog post and the [Great Expectations task](/api/latest/tasks/great_expectations.html).
```

Comment:
Do we want to create a copy of this blog on the website instead of medium?

Reply:
I will check with Alyssa